pages/api/setRepoConfig.ts

```diff
@@ -0,0 +1,28 @@
+import type { NextApiRequest, NextApiResponse } from 'next';
+import { isRepoIdentifier, type RepoIdentifier } from '../../types/repository';
+import { setRepoConfig } from '../../utils/db/repos';
+
+type SetRepoConfigReqBody = {
+	repo: RepoIdentifier,
+	configType: 'auto_assign' | 'comment',
+	value: boolean
+}
+
+const isSetRepoConfigReqBody = (x: any): x is SetRepoConfigReqBody => {
+	return x && isRepoIdentifier(x.repo) && (x.configType === "auto_assign" || x.configType === "comment") && typeof x.value === "boolean";
+}
+
+export default function handler(
```

Comment:
Use const and a better name than handler

Reply:
Fixed.